typings/node/node.d.ts

```diff
@@ -1292,7 +1292,7 @@ declare module "crypto" {
         update(data: any): void;
         sign(private_key: string, output_format: string): string;
     }
-    export function createVerify(algorith: string): Verify;
+    export function createVerify(algorithm: string): Verify;
```

Comment:
Ok, I'll just remove this one from the change then, and let you decide whether the def should be removed.

Thx !